### PR TITLE
Add changelog item for web-app-user-management

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: changelog/unreleased/enhancement-introduce-web-app-user-management [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Enhancement: Introduce web-app-user-management app
```
Comment (kulmann) [inline changelog/unreleased/enhancement-introduce-web-app-user-management:1]:
```suggestion
Enhancement: Introduce user-management app
```
User-facing the app name is "user-management" / "User management". `web-app-user-management` is only the package name.

File: changelog/unreleased/enhancement-introduce-web-app-user-management [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+We've added the app web-app-user-management with the following features
```
Comment (kulmann) [inline changelog/unreleased/enhancement-introduce-web-app-user-management:3]:
```suggestion
We've added the app "user-management" with the following features
```

**Review by kulmann — CHANGES_REQUESTED** [review]
Please add links of all the related pull requests

**Review by fschade — APPROVED** [review]
LGTM 🎸

**Review by kulmann — APPROVED** [review]

**Pull request merged**